ark/devnet/peers.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
Why would this be here? This is useless duplication of core specific configuration. Just because you can use 2 values of this file in your wallet doesn't mean we should make a package out of it.